services/alerta/service.go

```diff
@@ -17,13 +17,19 @@ import (
 type Service struct {
 	configValue atomic.Value
 	logger      *log.Logger
+	client      *http.Client
```

Comment:
The client field here should be an `atomic.Value` type and to be consistent we should call it  `clientValue`.